api/receivers/receivers.go

```diff
@@ -55,6 +69,19 @@ func getDefaultSMSConfig() webhookConfig {
 	}
 }
 
+func getDefaultPushoverConfig(userKey string) pushoverConfig {
+	return pushoverConfig{
+		SendResolved: true,
+		UserKey:      userKey,
+		Token:        viper.GetString("pushover_token"),
```

Comment:
Blir det riktig at plattformen skal eie applikasjonen som tokenet er knyttet til? Det betyr at vi betaler for alle alerts, men det gjør også at team ikke kan bruke Subscription API for å melde seg inn og ut av å få notifikasjon.

Bør se litt mer på dette, må bare bli litt mer kjent med plattformen.